src/screens/VisualTests/VisualTests.stories.tsx

```diff
@@ -1,8 +1,9 @@
 import { action } from "@storybook/addon-actions";
-import { expect } from "@storybook/jest";
+import { expect, jest } from "@storybook/jest";
```

Comment:
`jest` may or may not be unused?

Reply:
Would be nice if this was an error in linting or type checking..